SPECS-EXTENDED/rubygem-flexmock/rubygem-flexmock.spec

```diff
@@ -60,36 +62,26 @@ popd
 %check
 pushd .%{gem_instdir}
 
-tar xf %{SOURCE1}
-mv flexmock/test .
-
 ruby -Ilib:.:test \
 	-e 'Dir.glob("test/*_test.rb").each {|f| require f}'
 rspec test/rspec_integration/
 popd
 
-
 %files
-%license %{gem_instdir}/LICENSE.txt
-%dir	%{gem_instdir}
-%doc	%{gem_instdir}/[A-Z]*
-
+%license LICENSE.txt
+%dir %{gem_instdir}
 %{gem_libdir}
 %{gem_instdir}/rakelib/
-%exclude	%{gem_cache}
+%exclude %{gem_cache}
 %{gem_spec}
 
-%files	doc
-%{gem_instdir}/todo.txt
-%{gem_instdir}/doc/
+%files doc
 %{gem_docdir}/
 
 %changelog
-* Thu Feb 24 2022 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.3.6-8
```

Comment:
Please don't remove changelog entries.;) Especially since now the latest entry's release number doesn't match the spec's `Release` tag value.